doc/src/compute_count_type.rst

```diff
@@ -0,0 +1,88 @@
+.. index:: compute count/type
+
+compute count/type command
+====================
+
+Syntax
+""""""
+
+.. code-block:: LAMMPS
+
+   compute ID group-ID count/type mode
+
+* ID, group-ID are documented in :doc:`compute <compute>` command
+* count/type = style name of this compute command
+* mode = {atom} or {bond}
+  
+Examples
+""""""""
+
+.. code-block:: LAMMPS
+
+   compute 1 all count/type atom
+   compute 1 flowmols count/type bond
+
+Description
+"""""""""""
+
+Define a computation that counts the current number of atoms by atom
+type or the number of bonds by bond type.  The latter can be useful in
+reactive simulations where bonds are broken or created.
```

Comment:
Are there any fixes or commands that could lead to an unknown number of atoms of a given type in LAMMPS? Maybe fix evaporation? I think fix pour/deposit could do this too if it fails insertions.

Reply:
@jtclemm Good catch on fix bond/break.  I updated both the Howto broken bond and compute count/type pages.  See if the changes satisfy your concerns?

Reply:
I think the new edits to the broken bonds doc page look good. 

I guess if users want to track the number of bonds broken by fix bond/break, they can just rely on the output from its global vector. Maybe at some point in the future, if requested, this can be generalized to work across bond types.

Any thoughts on analogously listing relevant fixes which change the number of atoms as possible applications for the atom mode? Evaporation, pour, etc.